src/actors/snapshots/symbolicator__actors__symbolication__tests__minidump_windows.snap

```diff
@@ -154,7 +154,7 @@ stacktraces:
         trust: context
 modules:
   - debug_status: found
-    unwind_status: found
+    unwind_status: unused
```

Comment:
So perhaps this is wrong.  Line 42-52 of this snapshot shows a frame with `trust: cfi` so this should have been `found`.  @jan-auer could you confirm this?

Reply:
Confirmed.